.github/workflows/release.yml

```diff
@@ -136,6 +136,14 @@ jobs:
         if: ${{ !steps.release_info.outputs.release_info }}
         run: exit 1
 
+      - name: Verify upgrade handler exists
+        id: upgrade_handler
+        run: |
+          git fetch --all --tags
+          ver="$(git describe --tags --exact-match)"
```

Comment:
in theory, it's better to use`steps.release_info.outputs.release_tag` or `github.event.release.tag_name`
But here let's keep it as is because the same approach we can find above